### PR TITLE
Refactor strategies dict

### LINKED ISSUE #15: Expand strategies.py dict
Rather than just mapping fortran function names to axelrod-python class names, we should map to a dictionary.

That would allow us to expand the information we hold about each function without having to create new structures.

e.g.
```
strategies = {
   'ktitfortatc': {'axelrod-python class': 'TitForTat', 'stochastic': False, 'author': 'Unknown'}
}
```

### PR DESCRIPTION
**Pull request by meatballs**
closes #15

### CODE REVIEW

File: tests/test_player.py [hunk 1]
```diff
@@ -12,7 +12,11 @@
 def test_init():
     for strategy in all_strategies:
         player = Player(strategy)
-        assert player.classifier["stochastic"]
+        is_stochastic = classifiers[strategy]['stochastic']
+        if is_stochastic is not None and not is_stochastic:
```
Comment (drvinceknight) [inline tests/test_player.py]: this can be `if is_stochastic is False:`

File: src/axelrod_fortran/strategies.py [hunk 2]
```diff
@@ -1,73 +1,349 @@
...
-    'k89r': None,
-    'k90r': None,
-    'k91r': None,
-    'k92r': None,
-    'k93r': 'UnnamedStrategy',
-    'kpavlovc': 'WinStayLoseShift',
-    'krandomc': 'Random',
-    'ktf2tc': 'TitFor2Tats',
-    'ktitfortatc': 'TitForTat',
+classifiers = {
```
Comment (drvinceknight) [inline src/axelrod_fortran/strategies.py]: Suggestion: should we call this dictionary `information`? To avoid confusion with the `classifier` dictionary?

Comment (meatballs) [inline src/axelrod_fortran/strategies.py]: Naming things is difficult....

I considered both 'information' and 'details' but thought they were too vague and insufficiently descriptive. However, I also agree with the objection to 'classifiers'!

Comment (meatballs) [inline src/axelrod_fortran/strategies.py]: 'strategy_attributes' was another one that crossed my mind

Comment (meatballs) [inline src/axelrod_fortran/strategies.py]: a quick browse of the thesaurus suggests 'characteristics'

I think I like that one (possibly I just dislike it less than any other)

Comment (drvinceknight) [inline src/axelrod_fortran/strategies.py]: characteristics sounds fine to me :+1:

**Pull request merged**